src/OpenIddict.Core/OpenIddictModule.cs

```diff
@@ -1,24 +1,32 @@
-﻿using System;
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/openiddict/openiddict-core for more information concerning
+ * the license and the contributors participating to this project.
+ */
+
+using System;
+using System.Diagnostics;
 using Microsoft.AspNetCore.Builder;
 
 namespace OpenIddict {
     /// <summary>
-    /// Defines an OpenIddict module.
+    /// Represents an OpenIddict module.
     /// </summary>
+    [DebuggerDisplay("{Name,nq}")]
     public class OpenIddictModule {
```

Comment:
nit: this class can be made immutable since it's not supposed to change after `UseOpenIddict` it's been called